.ci/publish_sdist.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eu -o pipefail
+
+PASSWORD=$(echo $HACKAGE_PASSWORD | base64 --decode --ignore-garbage)
+
+SDIST=$(find $1-*.tar.gz | grep -v docs)
+DDIST=$(find $1-*.tar.gz | grep docs)
+
+set +u
+
+if [[ "$HACKAGE_RELEASE" == "yes" ]]; then
+    # Release tag set, upload as release
+    echo "Release!"
+    cabal upload --username=${HACKAGE_USERNAME} --password=${PASSWORD} ${SDIST} 
+    cabal upload --documentation --username=${HACKAGE_USERNAME} --password=${PASSWORD} ${DDIST} 
```

Comment:
You're not uploading a release, but release candidate.
Is that intentional?

For now I think prefer it this way.
After checking it we can manually publish a candidate as a release through the hackage webinterface.

But if that is what we want, the comments should reflect that.

Reply:
Yes, I was trying to be conservative as we can't unpublish. You're right that it should have been a comment.